docs/metrics.md

```diff
@@ -2,7 +2,7 @@
 
 [(Back to README)](../README.md)
 
-You can configure the Relay Proxy to export statistics and route traces to Datadog, Stackdriver, and Prometheus. To learn about the available settings for each of these options, read [Configuration](./docs/configuration.md).
+You can configure the Relay Proxy to export statistics and route traces to Datadog, Stackdriver, and Prometheus. To learn about the available settings for each of these options, read [Configuration](../configuration.md).
```

Comment:
Thanks for noticing the broken link, but your replacement link is also broken: `configuration.md` is in the same directory as `metrics.md`, so this should be a single period and not a double one.